file_download.php

```diff
@@ -197,6 +197,17 @@
 	http_content_disposition_header( $t_filename, $f_show_inline );
 }
 
+# Open image in the browser
+# https://www.thoughtco.com/mime-types-by-content-type-3469108
+if( stripos( $t_content_type, 'image/jpeg' ) !== false ||
+	stripos( $t_content_type, 'image/gif' ) != false ||
+	stripos( $t_content_type, 'image/tiff' ) != false ||
+	stripos( $t_content_type, 'image/bmp' ) != false ||
+	stripos( $t_content_type, 'image/svg+xml' ) != false ||
+	stripos( $t_content_type, 'image/png' ) != false ) {
```

Comment:
Instead of potentially calling stripos 6 times, I would suggest to use a single preg_match call instead, something like

```
~image\/(?:jpeg|gif|tiff|bmp|svg\+xml|png)~
```

Note that I didn't actually test the regex.